README.rst

```diff
@@ -16,7 +16,47 @@ The mixins support optional checksums of arbitrary columns, and asynchronous pro
 Documentation
 -------------
 
-The full documentation is at https://super-csv.readthedocs.org.
+Super CSV is a stand alone library that can be used for CSV management, both syncronous and async.
+
+Testing
+-------
+::
+
+    $make requirements
```

Comment:
nit: I don't think you need the '$' here

Reply:
I was using to symbolize command prompt, but will take them out.